packages/navigator/src/browser/navigator-widget.ts

```diff
@@ -32,4 +34,31 @@ export class FileNavigatorWidget extends FileTreeWidget {
         this.addClass(CLASS);
     }
 
-}
\ No newline at end of file
+    protected onAfterAttach(msg: Message): void {
```

Comment:
wonder whether we should pull it into `FileTreeWidget` to allow copying in other widgets, e.g. in the file dialog

Reply:
I think it's good here for now